### DISCUSSION TITLE
Search / filter on a data table

### DISCUSSION THREAD

**Discussion by ishah-twilio** [twilio-labs/paste#1715] [General] [answered]
Product: IoT > Super SIM > Data Insights
Background: Filter and Search data table

Do we have existing pattern or recommended best practices till pattern is release?
Does team have a feedback on split button pattern for search and filter on a data table? We like to test this in upcoming usability study on Aug 2nd.

<img width="1241" alt="search-filter-split button action menu" src="https://user-images.githubusercontent.com/69016180/127605383-86bdc83e-7ead-4e7c-9ba9-30c3a50c2138.png">

**Comment by SiTaggart** [comment]
Hi @ishah-twilio Something like this came up yesterday, and here is the resultant screenshot and feedback. https://github.com/twilio-labs/paste/discussions/1702#discussioncomment-1092729

Please do watch the recording to hear some feedback on the use of popovers for filters.

In the meantime we'll have an internal chat about how to display applied filters.

**Comment by kate-porter** [comment, marked as answer]
@corinne-nagel has also been working on a [Filter pattern](https://www.figma.com/file/D7VZ7jcwhAKOfOrjWLlpfO/Filtering-group-patterns?node-id=0%3A1) that is still a work-in-progress, but it might give you a good launching point until it's finalized.

We're definitely leaning towards creating a clear divide between searching and filtering, as you have in your design, but we've been looking at pulling search out of that filter section into its own space towards the top of the page.
